### PR TITLE
docs updated envs and Miniforge dir

### LINKED ISSUE #2386: docu fix name, pathes, wsgi setup
We need to use always mssenv, 

and we can install it by mamba

https://mss.readthedocs.io/en/stable/mswms.html#a-taste-of-wsgi

### PR DESCRIPTION
**Pull request by ReimarBauer**
**Purpose of PR?**:

Fixes #2386

### CODE REVIEW

File: docs/mswms.rst [hunk 1]
```diff
@@ -540,28 +540,41 @@ INSTANCE is a placeholder for your service name::
...
+ │         ├── LICENSE.txt
+ │         ├── locks
+ │         ├── man
+ │         ├── pkgs
+ │         ├── sbin
+ │         ├── share
+ │         ├── shell
+ │         ├── ssl
+ │         ├── x86_64-conda_cos6-linux-gnu
+ │         └── x86_64-conda-linux-gnu
```
Comment (matrss) [inline docs/mswms.rst]: This is indented too far compared to above.

File: docs/mswms.rst [hunk 2]
```diff
@@ -507,12 +507,12 @@ At current state we have to use pip to install mod_wsgi into the INSTANCE enviro
...
 Setup a /etc/apache2/mods-available/wsgi_express.conf::
 
-  WSGIPythonHome "/home/mss-demo/miniforge/envs/demo/"
+  WSGIPythonHome "/home/mss-demo/miniforge/envs/mssenv/"
 
 
 Setup a /etc/apache2/mods-available/wsgi_express.load::
 
-  LoadModule wsgi_module "/usr/lib/apache2/modules/mod_wsgi-py37.cpython-37m-x86_64-linux-gnu.so"
+  LoadModule wsgi_module "/usr/lib/apache2/modules/mod_wsgi-py310.cpython-310m-x86_64-linux-gnu.so"
```
Comment (joernu76) [inline docs/mswms.rst]: Maybe add a comment to replace the python version with the used one? This is terrible to maintain...

**Review by joernu76 — DISMISSED** [review]

File: docs/mswms.rst (hunk 2 above)
Comment (ReimarBauer) [reply docs/mswms.rst]: I think now it is better to understand how that works.

**Review by joernu76 — APPROVED** [review]

**Comment by ReimarBauer** [comment]
argh, double clicked without editing the squash log, sorry

**Pull request merged**